$NetBSD: patch-configure,v 1.1 2019/06/15 08:21:32 adam Exp $

Do not set -mmacosx-version-min.
Kill all of the overzealous optimization; if the user wants this,
there is devel/cpuflags and mk.conf.

--- configure.orig	2016-09-14 22:02:13.000000000 +0000
+++ configure
@@ -16589,10 +16589,6 @@ fi
 
 JACK_CORE_CFLAGS="$JACK_CORE_CFLAGS $JACK_LIBC_HELPER_FLAGS"
 
-case $build_os in
-     # we need weak linkage which appeared in 10.2, but lets ask for 10.4 anyway
-     darwin*) JACK_CORE_CFLAGS="$JACK_CORE_CFLAGS -mmacosx-version-min=10.4" ;;
-esac
 
 
 # Check whether --with-cpu-target was given.
@@ -16759,7 +16755,7 @@ rm -f core conftest.err conftest.$ac_obj
 
 $as_echo "#define x86 1" >>confdefs.h
 
-    COMMON_X86_OPT_FLAGS="-O3 -fomit-frame-pointer -ffast-math -funroll-loops"
+    COMMON_X86_OPT_FLAGS="-fomit-frame-pointer"
 
 
     if test x$with_cpu_target != x ; then
@@ -16794,7 +16790,7 @@ if test "x$enable_dynsimd" = xyes; then
 
 $as_echo "#define USE_DYNSIMD 1" >>confdefs.h
 
-		SIMD_CFLAGS="-O -msse -msse2 -m3dnow"
+		SIMD_CFLAGS=""
 
 fi
 
